go.mod

```diff
@@ -34,11 +34,10 @@ require (
 	k8s.io/klog/v2 v2.1.0
 	k8s.io/legacy-cloud-providers v0.18.3 // indirect
 	k8s.io/utils v0.0.0-20200603063816-c1c6865ac451 // indirect
-	knative.dev/pkg v0.0.0-20200528142800-1c6815d7e4c9
+	knative.dev/pkg v0.0.0-20200702222342-ea4d6e985ba0
```

Comment:
Shouldn't we also upgrade to a newer `knative.dev/pkg` (see replace section)?

Reply:
Done